clientgui/MainDocument.cpp

```diff
@@ -1712,10 +1712,10 @@ void CMainDocument::KillGraphicsApp(int pid) {
 
     if (g_use_sandbox) {
         snprintf(thePIDbuf, sizeof(thePIDbuf), "%d", pid);
-        argv[0] = "switcher";
-        argv[1] = "/bin/kill";
-        argv[2] =  "kill";
-        argv[3] = "-KILL";
+        argv[0] = (char *)"switcher";
```

Comment:
This is not a correct change. By default these strings are located in the application static resources and when you convert this to char* that is not constant, and then someone will device to change the value this pointer pointing at, there will be a crash. So I suggest not to make this change at all. Maybe one day we will consider rewriting this using std::string, and this issue will be solved then in a more sustainable way.